packages/samples/src/test/testApiCallbacks.ts

```diff
@@ -271,6 +271,113 @@ cl.AddCallback({
 })
 
 
+cl.AddCallback({
+    name: 'Callback With Results',
+    logic: async memory => {
+        const randomNumber = Math.round(Math.random() * 100)
+        memory.Set("myNumber", randomNumber)
+
+        if (randomNumber < 20) {
+            memory.Set('lowNumber', true)
+        }
+
+        if (randomNumber > 50) {
+            memory.Set('highNumber', true)
+        }
+
+        return randomNumber
+    },
+    render: async n => {
+        return `My random number is ${n}`
+    },
+    results: [
+        {
+            name: 'Number is 3',
```

Comment:
What do you think about just stringifying the value of this object rather than providing an explicit `name` ?  That would make the name never go stale, but makes the assumption that each stub will only care about changing a small number of properties.

Reply:
The name provides valuable semantic meaning.  Developer might tell the other user, use the callback result that called "Item is in Stock", regardless of what the entities are named or values are returned.  Similar to your comment this morning about a user looking at an weird ID, it's weird to look at stringified json object